src/ItspServices.pServer/ItspServices.pServer.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.3" />
+    <PackageReference Include="System.Data.SQLite" Version="1.0.111" />
```

Comment:
Maybe have a look at `Microsoft.Data.Sqlite` and check out if you want to use that instead.